presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreenState.kt

```diff
@@ -27,9 +31,11 @@ data class EventScreenState(
     val isStartDateTime: Boolean = false,
     val shouldOpenDropdown: Boolean = false,
     val alarmReminderItem: AlarmReminderItem = AlarmReminderItem.TEN_MINUTES,
-    val attendees: List<Attendee> = listOf(),
+    val attendees: List<Attendee> = mutableStateListOf(),
```

Comment:
Why mutablelist? You could have a immutablelist and just override it

If it's mutable, then it can be modified by the ui